client/app/lib/changeteam/controller.coffee

```diff
@@ -16,5 +16,9 @@ module.exports = class ChangeTeamController extends KodingListController
         return  if showError err
         groups = groups.filter (group) -> group.slug isnt 'koding'
         callback null, groups
+    options.noItemFoundWidget = new kd.CustomHTMLView
+      tagName  : 'p'
```

Comment:
I changed tag name of `noItemFoundWidget` to `p` in order to have `div`s for list items only. In this case I can use `:first-of-type` and `:last-of-type` to specify styles of first and last item in css